content/swagger/resources/measurement_families/routes/measurement_families.yaml

```diff
@@ -163,3 +163,46 @@ get:
       $ref: "#/responses/403Error"
     406:
       $ref: "#/responses/406Error"
+patch:
+  summary: Update/create several measurement families
+  operationId: "patch_measurement_families"
+  tags:
+    - Measurement family
+  x-versions:
+    - "Serenity"
+  description: This endpoint allows you to update and/or create several measurement families at once.
+  x-body-by-line: Contains several lines, each line is a measurement family in JSON standard format
```

Comment:
I don't think this is accurate. we send multiple measurement families in one json array. This is not the same mecanism as multiple JSON in the body. it's one JSON :/

Reply:
Ce serait quoi sur ce ?

Reply:
"Contains a json array. Each element of the JSON array is a measurement family in the standard format" ?

Reply:
```suggestion
  x-body-by-line: Contains a JSON array. Each element of the JSON array is a measurement family in the standard format
```